tests/integration_tests/modules/test_power_state_change.py

```diff
@@ -80,8 +80,7 @@ def test_poweroff(self, session_cloud: IntegrationCloud,
             "running 'init-local'",
             'config-power-state-change already ran',
         ]
-        assert ordered_items_in_text(lines_to_check, log), (
-            'Expected data not in logs')
+        assert verify_ordered_items_in_text(lines_to_check, log)
```

Comment:
```suggestion
        verify_ordered_items_in_text(lines_to_check, log)
```

(I'll apply this on merge.)